DatadogRUM/Tests/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -2830,4 +2830,63 @@ class RUMViewScopeTests: XCTestCase {
         XCTAssertEqual(metric.trackedViewStarts.count, 1)
         XCTAssertEqual(metric.trackedViewCompletes.count, 1)
     }
+
+    // MARK: - Has replay
+
+    func testViewUpdate_changesHasReplayFlag_onlyWhenReplayIsPresent() throws {
```

Comment:
nit, just a suggestion to make it clearer, but feel free to disregard

```suggestion
    func testViewUpdate_onceHasReplayIsTrueItRemainsTrue() throws {
```